src/cmd-generate-hashlist

```diff
@@ -0,0 +1,171 @@
+
+        self['meta'] = {}
+        if generator is None:
+            generator = 'coreos-assembler-' + metadata['coreos-assembler.container-image-git']['commit']  # noqa
+        self['meta']['generator'] = generator
+        if timestamp is None:
+            timestamp = datetime.datetime.utcnow().isoformat()
+        self['meta']['timestamp'] = timestamp
+        self['meta']['image'] = os.path.sep.join([
+            'builds', metadata['ostree-version'],
+            metadata['coreos-assembler.basearch'],
+            metadata['images']['qemu']['path']])
+
+        self['hashes'] = {}
+
+    def convert_file(self, file_path):
+        """
+        Converts the output from KeyLimes create_list.sh into
+        the HashList hash structure.
+
+        :param file_path: Path to the KeyLime list file
+        :type file_path: str
+        :raises: FileNotFoundError, IsADirectoryError, PermissionError
+        """
+        with open(file_path, 'r') as wp:
+            for line in wp.readlines():
+                sha256, path = line[:-1].split('  ')
+                self['hashes'][path] = [{'sha256': sha256}]
+
+    def write(self, output):
+        """
+        Renders the HashList structure to disk.
+
+        :param output: Where to write the file
+        :type output: str
+        :raises: FileNotFoundError, IsADirectoryError, PermissionError
+        """
+        with open(output, 'w') as out:
+            out.write(json.dumps(self, indent=4))
+
+
+def create_list(hashlist):
+    """
+    Starts up a VM and gets the list created.
+
+    :param hashlist: Iniitialized HashList instance
+    :type hashlist: HashListV1
+    :raises: CalledProcessError, PermissionError
+    """
+    try:
+        os.mkdir('hashmount')
+    except FileExistsError:
+        # It's OK if it already exists
+        pass
+    shutil.copy('/usr/lib/coreos-assembler/generate_list.sh', 'hashmount/generate_list.sh')
+    with tempfile.NamedTemporaryFile(mode='w') as fcct_yaml:
+            fcct_yaml.write(CREATE_HASH_FCCT)
+            fcct_yaml.flush()
+            ign_file = tempfile.mktemp()
+            subprocess.check_call(['/usr/bin/fcct', fcct_yaml.name, '-o', ign_file])
+            subprocess.check_call([
+                '/usr/bin/kola', 'qemuexec', '-i', ign_file,
+                '--bind-rw', '/srv/hashmount,/var/mnt/hashmount',
+                '--qemu-image', hashlist['meta']['image']])
+
+
+def main():
+    """
+    Main entry point.
+    """
+    parser = argparse.ArgumentParser(description=__doc__)
+    # parser.add_argument(
+    #     '-s', '--sign',
+    #     help='If set to a key a signature will be created as well')
+    parser.add_argument(
+        '-g', '--generator', help='Override the generator name')
+    parser.add_argument(
+        '-t', '--timestamp', help='Override the timestamp')
+    parser.add_argument(
+        '-r', '--release', help='Override the release')
+    parser.add_argument(
+        '-l', '--list',
```

Comment:
Given that the script is hardcoding the bind mount to the VM, should this really be configurable?